### DISCUSSION TITLE
Layout location error

### DISCUSSION THREAD

**Discussion by pathavyer** [Pilot-in/PiloPress#121] [Q&A]
Hi Pilopress dev team,

First of all, thank you for creating this plugin, I think its quite powerful and also great way to utilize tailwindcss. I'm in process of testing the plugin but as I create a test layout, there seem to be an error. please see screen shot below
![screenshot-tailwind-wordpress-lndo-site-wp-admin-admin-php-1613244200318](https://user-images.githubusercontent.com/45977961/107859485-a43c6e80-6dee-11eb-9434-6e3d6275f481.png)

**Comment by maelys-pilotin** [comment]
Hi @pathavyer!

Can you please share more details about your installation? Which versions of ACF Pro and ACFE are you using? Is it still occurring with latest versions of those plugins?
